src/scripts/lambda/README.md

```diff
@@ -0,0 +1,64 @@
+## Logging in
+
+SSH into one of the login nodes:
+
+```bash
+ssh ubuntu@192.222.61.240
+# OR: ssh ubuntu@192.222.60.249
+```
+
+Start or attach your own tmux session:
+
+```bash
+username=petew
+tmux new-session -A -s $username
+```
+
+Check that `/data/ai2/bin/` is in your `PATH`, and if not add it:
+
+```bash
+export PATH="/data/ai2/bin:$PATH"
+```
+
+Export your Beaker token to the `BEAKER_TOKEN` env variable:
+
+```bash
+export BEAKER_TOKEN=XXXX
+```
+
+## Initial setup
+
+We all login as the same user, so we need to be careful to use unique repo directories and virtual environment.
+
+```bash
+# Set your unique username (ideally use your Ai2/Beaker username).
+username=petew
+
+# Clone repo to unique directory.
+cd "/data/ai2/$username"
+git clone https://github.com/allenai/OLMo-core.git
+cd OLMo-core
+
+# Create unique virtual environment.
```

Comment:
I miss docker already haha

Reply:
Yea.. I figured we probably won't run into that 🤞 But we'll cross that bridge when we come to it.